templates/taxi/car_detail.html

```diff
@@ -3,13 +3,18 @@
 {% block content %}
   <h1>
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
 
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    {% if user in car.drivers.all %}
```

Comment:
Ensure that the `user` context variable is correctly passed to the template. This is necessary for the `{% if user in car.drivers.all %}` condition to work properly.